flink-tests/src/test/java/org/apache/flink/test/javaApiOperators/GroupCombineITCase.java

```diff
@@ -58,7 +58,7 @@
 */
 public class GroupCombineITCase extends MultipleProgramsTestBase {
 
-	public GroupCombineITCase(ExecutionMode mode) {
+	public GroupCombineITCase(TestExecutionMode mode) {
```

Comment:
I have committed this fix. If you rebase from master then you should get this change.